snuba/clickhouse/native.py

```diff
@@ -227,10 +227,6 @@ def __transform_result(self, result, with_totals: bool) -> Result:
 
     def execute(
         self,
-        # TODO: After we remove DictSqlQuery (that does all query processing for the
```

Comment:
I think this comment is not relevant anymore. There are plenty of places before this method where we need the formatted query for tracing and instrumentation or for caching. So Here we need to get a class that is closer to the formatted SQL.